### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by felddy**
# Lineage Pull Request: CONFLICT #

DANGER

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-packer.git`](https://github.com/cisagov/skeleton-packer.git)

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Clone the `lineage/skeleton` branch locally:

    ```console
    git clone --single-branch --branch lineage/skeleton git@github.com:cisagov/guacamole-packer.git
    ```

1. Review each of the following files looking for [conflict-resolution markers](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging#_basic_merge_conflicts):

    ```console
    .gitignore
    README.md
    terraform-build-user/versions.tf
    terraform-post-packer/main.tf
    terraform-test-user/versions.tf
    terraform/versions.tf
    ```

1. Resolve each of the conflicts and `add` your changes to the branch:

    ```console
    git add .gitignore README.md terraform-build-user/versions.tf terraform-post-packer/main.tf terraform-test-user/versions.tf terraform/versions.tf 
    ```

1. Once all the changes have been added, `commit` and `push` the changes back to
GitHub:

    ```console
    git commit --message="Resolved Lineage conflicts."
    git push
    ```

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Comment by jsf9k** [comment]
Something odd is going on here.  If I pull the latest code from origin and skeleton-packer and merge manually, there are only three commits that need to be added.

File: terraform-build-user/main.tf [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    aws                       = aws
+    aws.images-production-ami = aws.images-production-ami
+    aws.images-staging-ami    = aws.images-staging-ami
+    aws.images-production-ssm = aws.images-production-ssm
+    aws.images-staging-ssm    = aws.images-staging-ssm
+  }
+
+  ssm_parameters = ["/cyhy/dev/users", "/ssh/public_keys/*",
+  "/guacamole/postgres_username", "/guacamole/postgres_password"]
+  user_name = "test-guacamole-packer"
```
Comment (dav3r) [inline terraform-build-user/main.tf:14]: `test-guacamole-packer` --> `build-guacamole-packer`

File: terraform-post-packer/backend.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+terraform {
+  backend "s3" {
+    encrypt        = true
+    bucket         = "cisa-cool-terraform-state"
+    dynamodb_table = "terraform-state-lock"
+    profile        = "cool-terraform-backend"
+    region         = "us-east-1"
+    key            = "skeleton-packer/terraform-post-packer.tfstate"
```
Comment (dav3r) [inline terraform-post-packer/backend.tf:8]: `skeleton-packer` --> `guacamole-packer`

File: terraform-build-user/main.tf (hunk 1 above)
Comment (jsf9k) [reply terraform-build-user/main.tf]: See commit 2aedb22.

File: terraform-post-packer/backend.tf (hunk 2 above)
Comment (jsf9k) [reply terraform-post-packer/backend.tf]: See commit 2aedb22.

**Pull request merged**